src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -206,15 +208,4 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
 
     (indexDF, resolvedIndexedColumns, resolvedIncludedColumns)
   }
-
-  private def getPartitionColumns(df: DataFrame): Seq[String] = {
```

Comment:
This is no longer needed as  `FileBasedRelation` supports getting the partition schema.